powershell-adapter/Tests/TestClassResource/0.0.1/TestClassResource.psd1

```diff
@@ -0,0 +1,134 @@
+#
+# Module manifest for module 'TestClassResource'
+#
+# Generated by: migreene
+#
+# Generated on: 3/19/2024
+#
+
+@{
+
+# Script module or binary module file associated with this manifest.
+RootModule = '.\TestClassResource.psm1'
+
+# Version number of this module.
+ModuleVersion = '0.0.1'
+
+# Supported PSEditions
+# CompatiblePSEditions = @()
+
+# ID used to uniquely identify this module
+GUID = 'b267fa32-e77d-48e6-9248-676cc6f2327f'
+
+# Author of this module
+Author = 'migreene'
+
+# Company or vendor of this module
+CompanyName = 'Unknown'
+
+# Copyright statement for this module
+Copyright = '(c) migreene. All rights reserved.'
+
+# Description of the functionality provided by this module
```

Comment:
We should just delete all the generated comment/placeholders that aren't useful

Reply:
fixed